src/components/calcite-tip-manager/calcite-tip-manager.tsx

```diff
@@ -182,20 +187,57 @@ export class CalciteTipManager {
     this.nextTip();
   };
 
+  tipManagerKeyDownHandler = (event: KeyboardEvent): void => {
+    switch (event.key) {
+      case "ArrowUp":
+        event.preventDefault();
+      case "ArrowRight":
+        this.nextTip();
+        break;
+      case "ArrowDown":
+        event.preventDefault();
+      case "ArrowLeft":
+        this.previousTip();
+        break;
+      default:
```

Comment:
Is this default necessary?

Reply:
Short of it:
No. If you think it would be better if it wasn't there, I'll remove.

Long of it.
I remember learning at one point in the past that a switch statement should always have a default, so I added this one. I just checked and the MDN page says its optional, so I was probably recalling a best practice from back in the day (don't have need of switch statements that often - but it works well for this case). I think that probably had more to do with the switch statement handling core logic, where we'd be fine in this case if nothing at all happened.

Preference?

Reply:
Yeah, lets remove it since its empty.